StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/SA1613ElementParameterDocumentationMustDeclareParameterName.cs

```diff
@@ -50,7 +56,36 @@ public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics
         /// <inheritdoc/>
         public override void Initialize(AnalysisContext context)
         {
-            // TODO: Implement analysis
+            context.RegisterSyntaxNodeAction(this.HandleElement, SyntaxKind.MethodDeclaration);
```

Comment:
:bulb: Consider registering a handler for `XmlElement` and `XmlEmptyElement` instead.